### ISSUE TITLE
--expandTypography in token-transformer returns inconsistent lineHeight type

### ISSUE THREAD

**Issue by thomasmattheussen** [tokens-studio/figma-plugin#1610]
**Describe the bug**
When using `--expandTypography` in `token-transformer`, line height values receive `"type": "lineHeight"` instead of `"type": "lineHeights"` (plural)

**To Reproduce**
use token-transformer with the  `--expandTypography` option with this input
```
{
  "token": {
    "value": {
      "lineHeight": "24px"
    },
    "type": "typography"
  }
}
```
**Expected behavior**
It should be transformed into (notice the plural `lineHeights` type)
```
{
  "token": {
    "lineHeight": {
      "value": "24px",
      "type": "lineHeights"
    }
  }
}
```

**Comment by six7** [comment]
This was fixed with 0.30

**Comment by thomasmattheussen** [comment]
@six7 I'm still seeing this in 0.30 :/

**Comment by thomasmattheussen** [comment]
@six7 can this be reopened?

**Comment by six7** [comment]
Sure!

**Comment by six7** [comment]
Released in 0.33
